test-requirements.txt

```diff
@@ -7,4 +7,6 @@ vcrpy
 vcrpy-unittest
 openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 unittest2 # For shopinvader test_controller, which inherits component
-odoo-test-helper
+#odoo-test-helper
+# TMP dependency to include https://github.com/akretion/odoo-test-helper/pull/1
+git+git://github.com/simahawk/odoo-test-helper.git@master#egg=odoo-test-helper
```

Comment:
Use https://github.com/akretion/odoo-test-helper/pull/7 instead?

Reply:
I had issues w/ it and my version is working fine in other PRs here and on https://github.com/OCA/search-engine/pull/50
It's just that nor me nor @sebastienbeau had the time to dig further in what is not handled by my version or his.

Reply:
removed as new version has been released